website/static/js/fangorn.js

```diff
@@ -1675,7 +1675,7 @@ var FGToolbar = {
             );
         }
         //multiple selection icons
-        if(items.length > 1 && ctrl.tb.multiselected()[0].data.provider !== 'github' && ctrl.tb.options.placement !== 'fileview') {
+        if(items.length > 1 && ctrl.tb.multiselected()[0].data.provider !== 'github' && ctrl.tb.options.placement !== 'fileview' && !(ctrl.tb.multiselected()[0].data.provider === 'dataverse' && ctrl.tb.multiselected()[0].parent().data.version === 'latest-published') ) {
```

Comment:
Any special casing should be explained either with a comment or (even better) descriptive variable naming:

```js
var firstSelected = ctrl.tb.multiselected()[0];
var publishedDataverseSelected = (firstSelected.data.provider === 'dataverse' && 
    firstSelected.parent().data.version === 'latest-published') ;

if (items.length > 1 ... && !publishDataverseSelected)
```